frontend/src/metabase/dashboard/hooks/use-dashboard-url-params.ts

```diff
@@ -61,12 +61,11 @@ export const useDashboardUrlParams = ({
   });
 
   useEffect(() => {
-    const hashOptions = parseHashOptions(
-      location.hash,
-    ) as DashboardUrlHashOptions;
-
-    setFont(hashOptions.font ?? font);
-  }, [font, location.hash, setFont]);
+    const { font } = parseHashOptions(location.hash) as DashboardUrlHashOptions;
```

Comment:
Make the structure more similar to the other place we use `parseHashOptions`

https://github.com/metabase/metabase/blob/19cbeacb939ed9d6953b9de804f89ff041b5bdf1/frontend/src/metabase/public/hooks/use-embed-frame-options.ts#L8-L14